v2/src/log_stacktrace.c

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include <stddef.h>
+#include <stdbool.h>
+
+#include "windows.h"
+#include "winnt.h"
+#include "dbghelp.h"
+
+#include "c_logging/logging_stacktrace.h"
+
+static volatile LONG doSymInit = 0; /*0 = not initialized, 1 = initializing, 2= initialized*/
```

Comment:
> volatile LONG doSymInit = 0;  [](http://example.com/codeflow?start=6&length=30)

Once the BDF suggested using an anonymous union for these - /consider

Reply:
The value of doSymInit is never logged (where it makes sense to have it as a union). Nor do I expect huge debugging efforts around it.

Will leave it as is.

Reply:
Changed my mind and doing it anyway.